cellxgene_schema_cli/cellxgene_schema/utils.py

```diff
@@ -40,7 +40,7 @@ def map_ontology_term(dataframe, ontology_name, map_from_column, update_map):
     dataframe[column_name] = dataframe[column_name].cat.remove_unused_categories()
 
 
-def remove_deprecated_features(adata: ad.AnnData, deprecated: List[str]) -> ad.AnnData:
+def remove_deprecated_features(*, adata: ad.AnnData, deprecated: List[str]) -> ad.AnnData:
```

Comment:
Huh. The `*` marking the boundary between positional vs kwargs is new to me. Cool 🤙